test/quantization/test_quantized_nn_mods.py

```diff
@@ -850,5 +850,36 @@ def test_batch_norm3d(self):
         self.assertEqual(quant_ref.int_repr().numpy(), qy.int_repr().numpy(),
                          message="BatchNorm3d module API failed")
 
+    def test_layer_norm(self):
```

Comment:
Can we test this over qnnpack and FBGEMM? Qnnpack only tests the x86 path.
``` python
    @given(qengine=st.sampled_from(("qnnpack", "fbgemm")))
```
and using qengine as an argument to the test should do it.

Reply:
we don't have a mobile implementation for LayerNorm.  It might run correctly on mobile but it would be too slow to be useful.  Should we still test for it?  I'm flexible, more just curios on how we've done it in the past.